test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -76,6 +77,12 @@ const (
 	caNoScaleUpStatus      = "NoActivity"
 	caOngoingScaleUpStatus = "InProgress"
 	timestampFormat        = "2006-01-02 15:04:05 -0700 MST"
+
+	expendablePriority,lowPriority, highPriority = int32(-15), int32(100), int32(1000)
+	expendablePriorityClassName = "expendable-priority"
+	lowPriorityClassName = "low-priority"
```

Comment:
This class is never used in tests, do we need it?

Reply:
Removed